hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/VirtualBlobKey.java

```diff
@@ -141,11 +154,14 @@ public int getMinimumSupportedVersion() {
      * Verifies if the content from the buffer is equal to this virtual blob key.
      *
      * @param buffer The buffer with data to be compared with this key
-     * @param version The version of the data inside the buffer
      * @return If the content from the buffer has the same data as this instance
-     * @throws IOException If an I/O error occurred
      */
-    public boolean equals(final ByteBuffer buffer, final int version) throws IOException {
+    boolean equalsTo(final BufferedData buffer) {
+        return (type.ordinal() == (0xff & buffer.readByte())) && (entityNumCode == buffer.readInt());
+    }
+
+    @Deprecated(forRemoval = true)
```

Comment:
I do think it will be eventually removed, but probably not in the very next release. I'd expect JDB and PBJ data file formats to co-exist for some time, until all files in all networks are migrated from the former to the latter, which may take some time.

Let me leave `@Deprecated` annotation, but not for removal.